source/common/network/address_impl.h

```diff
@@ -102,6 +102,10 @@ class Ipv4Instance : public InstanceBase {
   struct IpHelper : public Ip {
     const std::string& addressAsString() const override { return friendly_address_; }
     bool isAnyAddress() const override { return ipv4_.address_.sin_addr.s_addr == INADDR_ANY; }
+    bool isUnicastAddress() const override {
```

Comment:
Done, also fixed an endianness bug that was apparent only after adding the tests, who would have thought :-)